clio-client/src/test/scala/org/broadinstitute/clio/client/dispatch/MoveGvcfSpec.scala

```diff
@@ -85,7 +85,7 @@ class MoveGvcfSpec extends BaseClientSpec {
     recoverToSucceededIf[IllegalArgumentException] {
       val command = MoveGvcf(
         key = testGvcfTransferV1Key,
-        destination = testGvcfCloudDestinationPath
+        destination = URI.create("gs://the-bucket/the/file.g.vcf.gz")
```

Comment:
Why isn't this one in TestData?

Reply:
Because, as you say, TestData wasn't a good idea.

Reply:
Agreed, and OK by me.